test/dotnet-build.Tests/dotnet-build.Tests.xproj

```diff
@@ -9,11 +9,13 @@
     <ProjectGuid>833ffee1-7eed-4f51-8dfd-946d48833333</ProjectGuid>
     <RootNamespace>Microsoft.DotNet.Tools.Builder.Tests</RootNamespace>
     <BaseIntermediateOutputPath Condition="'$(BaseIntermediateOutputPath)'=='' ">..\..\artifacts\obj\$(MSBuildProjectName)</BaseIntermediateOutputPath>
-    <OutputPath Condition="'$(OutputPath)'=='' ">..\..\artifacts\bin\$(MSBuildProjectName)\</OutputPath>
+    <OutputPath Condition="'$(OutputPath)'=='' ">..\..\artifacts\bin</OutputPath>
   </PropertyGroup>
-
   <PropertyGroup>
     <SchemaVersion>2.0</SchemaVersion>
   </PropertyGroup>
+  <ItemGroup>
+    <Service Include="{82a7f48d-3b50-4b1e-b82e-3ada8210c358}" />
```

Comment:
http://stackoverflow.com/questions/18614342/what-is-service-include-in-a-csproj-file-for